src/universal/utils/makeExternalLink.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react'
+
+const makeExternalLink = (copy, href) => (
```

Comment:
this looks like a react component, so let's make it a react component.
then we can do neat things to it, like say `HelpMenuLink = styled(ExternalLink)` to add an underline